test/runtests.jl

```diff
@@ -40,6 +40,9 @@ if !isdefined(Base.Test, Symbol("@test_nowarn"))
     end
 end
 
+
+tf_versioninfo()
```

Comment:
Can you add a comment that you're just testing it doesn't throw an error?

Reply:
Not just that it doesn't throw an error, it is also useful for debugging travis failures I think,
but sure,